internal/engine/procedures/procedural.go

```diff
@@ -0,0 +1,50 @@
+// package procedural ties together the different visitors to generate the plpgsql code for a procedure.
+package procedures
+
+import (
+	"github.com/kwilteam/kwil-db/core/types"
+	"github.com/kwilteam/kwil-db/internal/engine/ddl"
+	"github.com/kwilteam/kwil-db/internal/engine/procedures/clean"
+	"github.com/kwilteam/kwil-db/internal/engine/procedures/generate"
+	"github.com/kwilteam/kwil-db/internal/engine/procedures/typing"
+	"github.com/kwilteam/kwil-db/internal/parse/procedure"
+)
+
+// PreparedProcedures will prepare the plpgsql functions for all procedures in the schema.
```

Comment:
`PreparedProcedures` => `GeneratePLPGSQL`

I also notice that you started with `procedural` as the package name and went with `procedures`, but there are some imports a docs like at the top of this file using `procedural`